public/app/features/query/components/QueryEditorRenderer/AngularEditorRenderer.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { MutableRefObject, useEffect, useMemo, useRef } from 'react';
+import { DataSourceApi, EventBusSrv } from '@grafana/data';
+import { AngularComponent, getAngularLoader } from '@grafana/runtime';
+import { AngularQueryComponentScope } from '../QueryEditorRow';
+import { PanelModel } from 'app/features/dashboard/state/PanelModel';
+import { DashboardModel } from 'app/features/dashboard/state/DashboardModel';
+import { EditorRendererProps } from './types';
+import { getTimeSrv } from 'app/features/dashboard/services/TimeSrv';
+
+export function AngularEditorRenderer(props: EditorRendererProps): JSX.Element | null {
```

Comment:
@torkelo this seems to work. I have added a test page so you can play around with it but I think this will support all the use cases.

Reply:
Will remove this prior to merge to main.

Reply:
@mckn not seeing it being used from QueryEditorRow, that would help test this

Reply:
not seeing where  notifyAngularQueryEditorsOfData is handled https://github.com/grafana/grafana/blob/main/public/app/features/query/components/QueryEditorRow.tsx#L369. 

important for giving some angular editors access to query results and errors